lib/omniauth/rails_csrf_protection/token_verifier.rb

```diff
@@ -13,19 +12,15 @@ module RailsCsrfProtection
     # authenticity token, you can find the source code at
     # https://github.com/rails/rails/blob/v5.2.2/actionpack/lib/action_controller/metal/request_forgery_protection.rb#L217-L240.
     class TokenVerifier
-      include ActiveSupport::Configurable
-      include ActionController::RequestForgeryProtection
-
       # `ActionController::RequestForgeryProtection` contains a few
       # configurable options. As we want to make sure that our configuration is
-      # the same as what being set in `ActionController::Base`, we should make
-      # all out configuration methods to delegate to `ActionController::Base`.
-      config.each_key do |configuration_name|
-        undef_method configuration_name
-        define_method configuration_name do
-          ActionController::Base.config[configuration_name]
-        end
-      end
+      # the same as what being set in `ActionController::Base`, we use
+      # `class_attribute` to delegate to `ActionController::Base.config`.
+      #
+      # This approach replaces the deprecated ActiveSupport::Configurable usage.
+      class_attribute :config, default: ActionController::Base.config
+
+      include ActionController::RequestForgeryProtection
```

Comment:
note for maintainers: i'm not happy with this include coming after `class_attribute`, but the inclusion seems to require `config` to be available. fine to leave it this way if you're okay with it, but also open if you want another approach that can put the include up top

Reply:
Yeah, I think this is fine as we basically had `config` because we included `ActiveSupport::Configurable`.